app/src/main/java/App.kt

```diff
@@ -57,9 +57,6 @@ class App : Application() {
     }
 
     private fun initializeTimber() {
-        if (BuildConfig.DEBUG) {
-            Timber.plant(Timber.DebugTree())
-        }
```

Comment:
I want to set the tags manually instead of using the `DebugTree`. You may have noticed the `TimberTag` interface I have been slowly implementing, which is what this is about. I've removed the old `DebugTree` to remove the duplicate logging with the `ProductionTree` below and to see where I am missing tags as I work through the code.